.travis/build-pkgs

```diff
@@ -65,11 +66,11 @@ done
 
 printf '\n----\n'
 if [ -n "$successful_pkgs" ]; then
-	print -s1 -c2 "Successfully build packages:$successful_pkgs\n"
+	print -s1 -c2 "Successfully build packages: $successful_pkgs\n"
```

Comment:
There should not be a space, because if `$successful_pkgs` is not empty, then it always starts with a space. See [line 44](https://github.com/7heo/aports/blob/d418c0d19d1d74a409d11b47fbef900c3d6d13f7/.travis/build-pkgs#L44) and [line 59](https://github.com/7heo/aports/blob/d418c0d19d1d74a409d11b47fbef900c3d6d13f7/.travis/build-pkgs#L59). The same for the `$failed_pkgs` variable.